ale/drivers/base.py

```diff
@@ -1207,7 +1223,32 @@ def _sensor_position(self):
         """
         if not hasattr(self, "_inst_position_table"):
             self._init_tables()
-        return self._inst_position_table.get('Positions', None)
+
+        inst_positions_times = np.linspace(self._inst_position_table["Times"][0],
+                                           self._inst_position_table["Times"][-1],
+                                           int(self.number_of_ephemerides))
+
+        # interpolate out positions
+        if len(self._inst_position_table["Times"]) < 2:
+            time_0 = self._inst_position_table["Times"][0]
+            position_0 = self._inst_position_table["Positions"][0]
+            velocity_0 = self._inst_position_table["Velocities"][0]
+            coefs = np.asarray([position_0 - time_0*velocity_0,
+                                velocity_0])
+            positions = np.polynomial.polynomial.polyval(inst_positions_times, coefs)
```

Comment:
Is this capability that is also in ALE? If so, should we be eating our own dog food?

Reply:
ALE is not currently able to do vectorized interpolation like scipy is.

Reply:
If the capability isn't there now, should this be in the C++ part of ale that handles interpolation? (This may be part of a broader question about how I'm not sure how the C++ part of ale fits in with the current architecture anymore.)

Reply:
I'd rather not add vectorized interpolation over Numpy arrays to ALE. It's already in scipy and from the c++ end, we're not going to be interpolating over Numpy arrays. The only benefit would be not using scipy, but it turns out scipy has a really good rotation module we're using else where.

Reply:
Works here. I would note that getting C access to a bumpy array is cake. The use of spicy elsewhere is a stronger justification for keeping this here.